### PR TITLE
Add Annotations for graph

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
* One graph consist of edges and vertexes

* @Graph specify the domain class as one graph

* @EdgeSet specify one field in graph for all edges

* @VertexSet specify one field in graph for all vertexes

Signed-off-by: Pan Li <panli@microsoft.com>

### CODE REVIEW

File: src/main/java/com/microsoft/spring/data/gremlin/annotation/EdgeSet.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.annotation;
+
+import org.springframework.data.annotation.Persistent;
+
+import java.lang.annotation.*;
+
+/**
+ * Specifies the field as EdgeSet of graph.
```
Comment (yungezz) [inline src/main/java/com/microsoft/spring/data/gremlin/annotation/EdgeSet.java:13]: put those above? and consistent in all files

**Review by yungezz — APPROVED** [review]

File: src/main/java/com/microsoft/spring/data/gremlin/annotation/EdgeSet.java (hunk 1 above)
Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/spring/data/gremlin/annotation/EdgeSet.java:13]: seems mongodb and neo4j use this style here. Do we need to align with them?

Comment (yungezz) [reply src/main/java/com/microsoft/spring/data/gremlin/annotation/EdgeSet.java:13]: no need. actually we don't add @author that in codes, except license disclaimer. but add <developer> in pom.xml, you could check azure-spring-boot

**Review by sophiaso — APPROVED** [review]

File: src/main/java/com/microsoft/spring/data/gremlin/annotation/EdgeSet.java (hunk 1 above)
Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/spring/data/gremlin/annotation/EdgeSet.java:13]: OK, copy that.

**Pull request merged**